.github/workflows/demo.yml

```
@@ -0,0 +1,152 @@
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get Atmos version
+        id: get-version
+        run: |
+          VERSION=$(curl -s https://api.github.com/repos/cloudposse/atmos/releases/latest | jq -r .tag_name)
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: List all .tape files
+        id: list-tapes
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:23: Double quote to prevent globbing and word splitting

(shellcheck)